src/extract.ts

```diff
@@ -363,7 +363,7 @@ function extractCatch2Result(output: string): BenchmarkResult[] {
             );
         }
 
-        const range = '+/- ' + stdDev[1].trim();
+        const range = '± ' + stdDev[1].trim();
```

Comment:
Would you tell me why this change is necessary? I'm not against `±`, but I wonder what is the better than `+/-`.

Reply:
We have a lot of benchmarks, which means the names of those benchmarks are a bit longer. This reduces the width of the table that's commented on the commit which means it looks nicer.

Reply:
OK, that makes sense.